packages/renderer/src/lib/feedback/feedbackForms/DevelopersFeedback.svelte

```diff
@@ -16,16 +16,20 @@ import type { FeedbackProperties } from '/@api/feedback';
 
 import WarningMessage from '../../ui/WarningMessage.svelte';
 
-// feedback of the user
-let smileyRating = 0;
-let tellUsWhyFeedback = '';
-let contactInformation = '';
+interface Props {
+  onCloseForm?: () => void;
+}
 
-$: hasFeedback =
+// feedback of the user
+let smileyRating = $state(0);
+let tellUsWhyFeedback = $state('');
+let contactInformation = $state('');
+let hasFeedback = $derived(
   (tellUsWhyFeedback && tellUsWhyFeedback.trim().length > 4) ||
-  (contactInformation && contactInformation.trim().length > 4);
+    (contactInformation && contactInformation.trim().length > 4),
+);
 
-export let onCloseForm = () => {};
+let { onCloseForm }: Props = $props();
```

Comment:
I think it needs to have a default value if not specified

```suggestion
let { onCloseForm = () => {} }: Props = $props();
```